indico/modules/rb/blueprint.py

```diff
@@ -32,6 +32,9 @@
 _bp.add_url_rule('/api/stats', 'stats', misc.RHStats)
 _bp.add_url_rule('/api/map-areas', 'map_areas', misc.RHMapAreas)
 _bp.add_url_rule('/api/equipment', 'equipment_types', misc.RHEquipmentTypes)
+_bp.add_url_rule('/api/tos', 'tos', misc.RHTos)
```

Comment:
I'd go for `RHTOS` or `RHTermsOfService`; I think mixed-cases acronyms look weird 🙈